### PR TITLE
Packaging

### PR DESCRIPTION
**Pull request by ruimbribeiro**

### CODE REVIEW

File: example_consumer/docker-compose-sqs-consumer.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    volumes:
+      - '.:/data/www'
+    environment:
+      - 'AWS_ACCESS_KEY_ID=x'
+      - 'AWS_SECRET_ACCESS_KEY=x'
+      - 'AWS_SQS_SNS_REGION=local'
+      - 'AWS_SQS_SNS_ENDPOINT_URL=http://herbie-aws-sns-sqs:4100'
+    networks:
+      - 'sandbox_herbie-network'
+
```
Comment (tobi812) [inline example_consumer/docker-compose-sqs-consumer.yml:17]: We could think about adding an sqs/sns container here for testing.

File: aws_sns_sqs_adapter/django_settings.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import environ
+
+env = environ.Env()
+
+AWS_ACCESS_KEY_ID = env.str('AWS_ACCESS_KEY_ID', " ")
+AWS_SECRET_ACCESS_KEY = env.str('AWS_SECRET_ACCESS_KEY', " ")
+AWS_SQS_SNS_REGION = env.str('AWS_SQS_SNS_REGION', " ")
+AWS_SQS_SNS_ENDPOINT_URL = env.str('AWS_SQS_SNS_ENDPOINT_URL', " ")
```
Comment (tobi812) [inline aws_sns_sqs_adapter/django_settings.py]: Let's stick to single quotes here.

File: aws_sns_sqs_adapter/django_settings.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+print(INSTALLED_APPS)
```
Comment (tobi812) [inline aws_sns_sqs_adapter/django_settings.py]: Do we need this line here?

**Review by tobi812 — CHANGES_REQUESTED** [review]
Just found a couple of minor things, besides that it looks supernice :+1:

File: aws_sns_sqs_adapter/publisher/aws_sns_topic_publisher.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        )
+
+    def get_name(self) -> str:
+        return 'aws_sns_sqs'
+
+    def send_message(self, message: Message):
+        serializer = message.get_serializer()
+
+        topic = serializer.data['type']
+        # Topics are immutable to it's safe to call create as the already existing topic will be returned
```
Comment (tobi812) [inline aws_sns_sqs_adapter/publisher/aws_sns_topic_publisher.py:34]: :+1:

**Review by tobi812 — APPROVED** [review]
LGTM, nice job :+1:

File: example_consumer/docker-compose-sqs-consumer.yml (hunk 1 above)
Comment (ruimbribeiro) [reply example_consumer/docker-compose-sqs-consumer.yml:17]: The idea was that this container would be in the sandbox docker-compose as we need to publish things from there, that's why I added the sandbox docker network here as an _external one_ 

```
networks:
  sandbox_herbie-network:
    external: true
```

Comment (tobi812) [reply example_consumer/docker-compose-sqs-consumer.yml:17]: aha makes sense :+1:

**Review by tobi812 — APPROVED** [review]
Wonderful: LGTM :+1:

**Pull request merged**